x-pack/plugins/enterprise_search/public/applications/app_search/app_logic.ts

```diff
@@ -16,6 +16,7 @@ export interface IAppActions {
 }
 
 export const AppLogic = kea<MakeLogicType<IAppValues, IAppActions>>({
+  path: ['enterprise_search', 'app_search', 'app'],
```

Comment:
I added pathing to our Kea Logic files.

This has two benefits.

1. In tests, we can initialize values in a logic to whatever a test needs. I do that by using the `defaults` option in `resetContext`: https://github.com/elastic/kibana/blob/b0fcf4a24ab33c8b39fc68a2162994c064b20eab/x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_logic.test.ts#L21-L33

2. When debugging using Redux DevTools, you can now see namespaced stores, rather than inline stores.

<img width="1534" alt="namespaces" src="https://user-images.githubusercontent.com/1427475/93359135-417b8280-f810-11ea-8132-4b3b18a1c301.png">

https://kea.js.org/docs/api/context/#resetcontext

Reply:
This is a great pattern. Thanks for adding it!

Reply:
Whoah TIL